### ISSUE TITLE
Calling ncu.run with cwd option alters the current working directory.

### ISSUE THREAD

**Issue by aureooms** [raineorshine/npm-check-updates#849]
## Steps to Reproduce
```js
process.chdir('/foo');
await ncu.run({cwd: '/bar'});
console.debug(process.cwd());
```

## Current Behavior
Outputs `/bar`.

## Expected Behavior
Outputs `/foo`.

**Comment by aureooms** [comment]
To circumvent the issue I use the `packageFile` option instead. Could this lead to problems?

**Comment by raineorshine** [comment]
Definitely a bug! Thanks for reporting.

**Comment by raineorshine** [comment]
Fixed and published in `v11.2.2`

**Comment by aureooms** [comment]
Thanks!
